### PR TITLE
update for JL3 with 'jupyterlab.upgrade_extension'

### PR DESCRIPTION
**Pull request by ahgraber**
Ran the "cheater" [jupyterlab migration script](https://jupyterlab.readthedocs.io/en/stable/extension/extension_migration.html) `python -m jupyterlab.upgrade_extension .` and a quick local install.  Seems to work for me.  

Sending PR to hopefully reduce your dev time :)

### CODE REVIEW

**Comment by mlshapiro** [comment]
oh man thanks so much. this has fallen so far behind on things I can pay attention to! I will merge this as soon as I can

File: _temp_extension/.eslintignore [hunk 1]
```diff
@@ -0,0 +1,5 @@
+node_modules
```
Comment (jtpio) [inline _temp_extension/.eslintignore:1]: These `_temp_extension/*` files should normally not be committed to the repo.

And could be added to the main `.gitignore`.

Comment (ahgraber) [reply _temp_extension/.eslintignore]: Thanks, I think I fixed it in the new commit...

**Comment by marctorsoc** [comment]
> oh man thanks so much. this has fallen so far behind on things I can pay attention to! I will merge this as soon as I can

what's the status of this?

**Comment by ahgraber** [comment]
bump - LMK if there's anything further I can do to support.

**Comment by hstarmans** [comment]
would appreciate if this can be accepted, tried to build and compile from source, using jlpm etc, but ran into lots of issues.

**Comment by marctorsoc** [comment]
maybe we need to cite the reviewer :) @jtpio

**Comment by falconair** [comment]
Hi, looks like the last comment was a few months ago. I would love to have my students install a linter and have it check their code as a way to improve their coding habits. If this project isn't being updated anymore (last commit is from a year and a half ago), is there a good alternative for the latest versions of jupyter lab?

**Comment by mlshapiro** [comment]
Sorry all, finally getting around to this - hope to push today. I really appreciate the help -

**Comment by mlshapiro** [comment]
updated to 0.7 with support for JupyterLab 3.x

**Pull request closed without merging**